.gitlab-ci.yml

```diff
@@ -0,0 +1,118 @@
+# see https://docs.gitlab.com/ce/ci/yaml/README.html for all available options
+
+variables:
+  SCHEDULER_PARAMETERS: "-J ArborX_CI -W 0:30 -nnodes 1 -P CSC333 -alloc_flags gpumps"
```

Comment:
Just a general question about it: where is this all executed? On batch node? Are both jsrun commands result in the same execution node?

Reply:
This is for the `batch` nodes. The `nobatch` stages don't need this AFAICT.
I observed both "run" stages to be executed on the same compute node so they should not be run in parallel.

Reply:
I don't see `smt1` or any similar options. We should probably add it.